.gitignore

```diff
@@ -3,3 +3,5 @@ dist
 lib
 module
 node_modules
+.idea/
+*.iml
```

Comment:
Would suggest to keep those in a personal `.gitignore`.

Reply:
Reverted this change.